packages/react/src/components/Code/Code.test.tsx

```diff
@@ -15,10 +15,18 @@ beforeEach(() => {
       disconnect: sandbox.stub()
     };
   });
+
+  // Mock the clipboard API
+  Object.assign(navigator, {
```

Comment:
This globally overrides `navigator.clipBoard` so could affect other tests. You might not need this `Object.assign` once #1603 gets merged as this gets stubbed in `setupTests` and then you should be able to easily set a jest spy:

https://github.com/dequelabs/cauldron/blob/dc285a99a27a70b7b1bf62a8066d2acd6317f297/packages/react/src/setupTests.ts#L24-L32

https://github.com/dequelabs/cauldron/blob/dc285a99a27a70b7b1bf62a8066d2acd6317f297/packages/react/src/components/CopyButton/CopyButton.test.tsx#L38-L41